src/eedl/image.py

```diff
@@ -8,16 +8,16 @@
 import ee
 import fiona
 import rasterstats
-from ee import EEException
+# from ee import EEException
 
 from . import google_cloud
 from . import mosaic_rasters
 
-try:
-	ee.Initialize()
-except EEException:
-	ee.Authenticate()
-	ee.Initialize()
+# try:
```

Comment:
Hey Adam, remind me where all of this code was moved to? We need it to trigger at some point. It shouldn't be here for our tests to run right now, but does need to happen at the relevant moment, so it shouldn't just be commented out.

Reply:
It hasn't been moved yet. Should we have an initialize method or something. ie have to run eedl.initalize() if you want to do anything with Earth Engine?

Reply:
Yes, exactly - something like that needs to be called near the top of the `export` method on the `Image` class

Reply:
Do you think the ee stuff should just go into __init__ or into a separate function (initalize)